packages/snippet-bot/test/snippet-bot.ts

```diff
@@ -138,6 +138,81 @@ describe('snippet-bot', () => {
       diffRequests.done();
     });
 
+    it('quits early for normal labels', async () => {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const payload = require(resolve(
+        fixturesPath,
+        './pr_event_label_ignored'
+      ));
+      const requests = nock('https://api.github.com')
+        .get(
+          '/repos/tmatsuo/repo-automation-bots/contents/.github%2Fsnippet-bot.yml'
+        )
+        .reply(200, config);
+
+      await probot.receive({
+        name: 'pull_request',
+        payload,
+        id: 'abc123',
+      });
+
+      requests.done();
+    });
+
+    it('responds to snippet-bot:refresh label', async () => {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const changedFiles = require(resolve(fixturesPath, './pr_files_added'));
+      const payload = require(resolve(fixturesPath, './pr_event_label_added'));
+      const blob = require(resolve(fixturesPath, './failure_blob'));
+
+      const requests = nock('https://api.github.com')
+        .get(
+          '/repos/tmatsuo/repo-automation-bots/contents/.github%2Fsnippet-bot.yml'
+        )
+        .reply(200, config)
+        .delete(
+          // For removing the label.
+          '/repos/tmatsuo/repo-automation-bots/issues/14/labels/snippet-bot%3Arefresh'
+        )
```

Comment:
thanks for further fleshing out the tests 👍

Reply:
ack